aws-lc-fips-sys/CMakeLists.txt

```diff
@@ -52,8 +52,15 @@ endforeach ()
 if (BORINGSSL_PREFIX)
     if (MSVC)
         set(TARGET_PREFIX "${BORINGSSL_PREFIX}")
+        set_my_target_properties(IMPORT_PREFIX ${TARGET_PREFIX})
     else()
         set(TARGET_PREFIX "lib${BORINGSSL_PREFIX}")
     endif()
-    set_my_target_properties(PREFIX ${TARGET_PREFIX})
+    set_my_target_properties(PREFIX ${TARGET_PREFIX} )
 endif ()
+
+if (BUILD_LIBSSL)
+    install(TARGETS rust_wrapper crypto ssl DESTINATION .)
+else()
+    install(TARGETS rust_wrapper crypto DESTINATION .)
+endif()
```

Comment:
Doesn't the build script have some logic to also copy the `include` headers into this path? I guess what does this produce, cause I don't think it inherits the install artifact behavior defined in AWS-LC right?

Reply:
* The DLLs were not being found at runtime unless they were in the `build/artifacts`.
    * This is the we set for: `cargo:rustc-link-search=native=`).
* The DLLS were only being placed `build/artifacts` when I use the "install" target.
    * If I left off the `DESTINATION` parameters, it would place the DLLs under `build/artifacts/bin`, where they weren't being found.